### ISSUE TITLE
App works fine but when hydration is enabled I get an error "failed"

### ISSUE THREAD

**Issue by dunctem** [phonegap/build#524]
Any idea why this happens?

**Comment by TatsuUkraine** [comment]
I have same issue

**Comment by wildabeast** [comment]
Please provide:
- platform
- phonegap version
- App ID
- are you seeing "failed" in the build, or when running the app?

**Comment by TatsuUkraine** [comment]
- Android
- last
- currently I don't use hydration
- running app. Black screen with message like "Whoops..." and red alert on next row with just message "failed"

**Comment by TatsuUkraine** [comment]
Also I have build fail for Android when I'm trying to use whitelist plugin in app with hydration turned on, in logs I get message "whitelist plugin already loaded"

**Comment by wildabeast** [comment]
App ID?

**Comment by TatsuUkraine** [comment]
I removed app with this issue and currently I don't have app with hydration

**Comment by dunctem** [comment]
Yeah I gave up as well, I will build my app with hydration enabled and comment if it works or not.

**Comment by dunctem** [comment]
Ok, I tried again and have the same problem on a different device

App ID: 2125639

Currently only testing with android.

The app works perfectly fine until I turn hydration ON and reinstall, then is gives me an error screen and I can see a red bar but no error information.

Device name: SGH-T399N
Android Version 4.4.2

**Comment by Aloxord** [comment]
i have right now the same problem, the hydration break the app
